clang/test/CodeGenSYCL/fpga-attr-do-while-loops.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang++ -fsycl-device-only -fintelfpga -S %s -o - | FileCheck %s
+// RUN: %clang -fsycl-device-only -fintelfpga -S %s -o - | FileCheck %s
```

Comment:
Should this be `%clang_cc1` instead? all the other tests are using %clang_cc1.
Using %clang looks confusing , as this is apparently not c code?

Reply:
Using of `%clang` is intentional, since the original fix was made to the driver.

Reply:
How does `-fsycl-device-only` work with `%clang`?   Isn't it only a cc1 option?   Perhaps that should be changed to `-fsycl`.

Reply:
Right, sorry.   I mistook it for `-fsycl-is-device`.